ecmascript/transforms/compat/src/es2017/async_to_generator.rs

```diff
@@ -211,6 +213,35 @@ impl Fold for Actual {
                 );
             }
 
+            Expr::Assign(assign_expr) => {
+                // flag if expression is assignment to prototype chain should not try to bind
+                // this context
+                match &assign_expr.left {
+                    PatOrExpr::Pat(pat) => match &**pat {
+                        Pat::Expr(pat_expr) => match &**pat_expr {
+                            Expr::Member(MemberExpr {
+                                obj: ExprOrSuper::Expr(ex),
+                                ..
```

Comment:
I think `computed: false` is required.

Reply:
added check for the property (https://github.com/swc-project/swc/pull/2667/files#diff-3e3528801f0642de9a7612e339d4ba8958e38dac96a90d6c972aa8f49769a363R226), is this correct understanding?

Reply:
You can use like 

```rust

match &**pat_expr {
     Expr::Member(MemberExpr {
         obj: ExprOrSuper::Expr(ex),
         computed: false,
         ..
     }) => {}
}
```

instead

Reply:
oh thansk. :facepalm: updated!